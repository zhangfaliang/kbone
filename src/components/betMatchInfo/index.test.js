import toJson from 'enzyme-to-json';
import { mount, render } from 'enzyme';
import BetMatchInfo from './index';

describe('BetMatchInfo', async () => {
  it('base', () => {
    expect.assertions(1);
    const wrapper = render(<BetMatchInfo />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
