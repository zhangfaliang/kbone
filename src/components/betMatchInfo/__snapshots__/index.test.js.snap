// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BetMatchInfo base 1`] = `
<div
  class="wrap"
>
  <div
    class="nameOrScore"
  >
    <span
      class="name"
    >
      Villacher Sv
    </span>
  </div>
  <div
    class="nameOrScore"
  >
    <span
      class="name"
    >
      Greifenburg
    </span>
  </div>
  <div
    class="bottom"
  >
    <span
      class="live"
    >
      LIVE
    </span>
    <span
      class="time"
    >
      4th set 10'
    </span>
  </div>
</div>
`;
